components/Molecule/mainbanner.html

```diff
@@ -1,58 +1,107 @@
 <div>
```

Comment:
Needs the component info:

`<div class="component"  title="main-banner" version="1.0">`